config-model/src/main/resources/schema/deployment.rnc

```diff
@@ -73,3 +74,18 @@ Delay = element delay {
 Parallel = element parallel {
     Region*
 }
+
+
+EndpointRegion = element region {
+    text
+}
+
+Endpoint = element endpoint {
+    attribute id { xsd:string }? &
+    attribute container-id { xsd:string } &
+    EndpointRegion*
+}
+
+Endpoints = element endpoints {
+    Endpoint*
```

Comment:
Since `<endpoints/>` is the same as not declaring the element at all, I think we should require `Endpoints` to contain at least one `Endpoint`. I believe the syntax for this is `Endpoint+`.